docs/topics/clusterdefinitions.md

```diff
@@ -248,6 +248,7 @@ The reason for the unsightly base64-encoded input type is to optimize delivery p
 | kube-apiserver | true | 1 per master node | The Kubernetes API server validates and configures data for the api objects which include pods, services, replicationcontrollers, and others. The API Server services REST operations and provides the frontend to the cluster's shared state through which all other components interact. Official docs [here](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/). |
 | kube-scheduler | true | 1 per master node | The Kubernetes scheduler is a policy-rich, topology-aware, workload-specific function that significantly impacts availability, performance, and capacity. The scheduler needs to take into account individual and collective resource requirements, quality of service requirements, hardware/software/policy constraints, affinity and anti-affinity specifications, data locality, inter-workload interference, deadlines, and so on. Official docs [here](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/). |
 | kube-addon-manager | true | 1 per master node | Addon manager provides a standard way to deliver additional Kubernetes componentry. The addons supported by AKS Engine (documented [above](#addons)) are installed into the cluster using Addon manager. Official docs [here](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager). |
+| cluster-init | false | n/a | This is a "backdoor" interface for delivering manual Kubernetes resource configuration that is appropriate for tight coupling to the ARM deployment specification. For example, if you are using aks-engine in an automated pipeline and you can scale up a cluster more quickly by loading a Kubernetes specification immediately after bootstrapping the control plane, you may use this interface by base64 encoding a yaml representation of that spec. The base64-encoded data string will be decoded at cluster creation time into a yaml file on the first master VM at the `/opt/azure/containers/cluster-init.yaml` filepath, and then loaded into the cluster via `kubectl apply -f`. See[here](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply) for more documentation on how to appropriate specify your source yaml for `kubectl apply` before base64 encoding it. Note: the `cluster-init` spec will be ignored during `aks-engine upgrade` operations, nor will it be re-invoked following any node pool scale operations. |
```

Comment:
I think this could be shortened. Also it should be "base64-encoded".

| cluster-init | false | n/a | This is an interface to deliver Kubernetes resource configuration that may be tightly coupled to an ARM deployment. For example, if you use AKS Engine in an automated pipeline and you can scale up a cluster quickly by loading a Kubernetes specification immediately after bootstrapping the control plane, give the base64-encoded YAML representation of that spec to `cluster-init`. That data is decoded at cluster creation time to the path `/opt/azure/containers/cluster-init.yaml` on the first master VM, and then loaded into the cluster via `kubectl apply -f`. See[`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply) for more documentation on how to specify the source YAML. Note: the `cluster-init` spec is ignored during `aks-engine upgrade` and `aks-engine scale`operations.